airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/JdbcSource.java

```diff
@@ -24,203 +24,25 @@
 
 package io.airbyte.integrations.source.jdbc;
 
-import static org.jooq.impl.DSL.currentSchema;
-import static org.jooq.impl.DSL.field;
-import static org.jooq.impl.DSL.table;
-
 import com.fasterxml.jackson.databind.JsonNode;
-import io.airbyte.commons.json.Jsons;
-import io.airbyte.commons.lang.Exceptions;
-import io.airbyte.commons.resources.MoreResources;
-import io.airbyte.db.DatabaseHelper;
 import io.airbyte.integrations.base.IntegrationRunner;
 import io.airbyte.integrations.base.Source;
-import io.airbyte.protocol.models.AirbyteCatalog;
-import io.airbyte.protocol.models.AirbyteConnectionStatus;
-import io.airbyte.protocol.models.AirbyteConnectionStatus.Status;
-import io.airbyte.protocol.models.AirbyteMessage;
-import io.airbyte.protocol.models.AirbyteMessage.Type;
-import io.airbyte.protocol.models.AirbyteRecordMessage;
-import io.airbyte.protocol.models.AirbyteStream;
-import io.airbyte.protocol.models.CatalogHelpers;
-import io.airbyte.protocol.models.ConnectorSpecification;
-import io.airbyte.protocol.models.Field;
-import io.airbyte.protocol.models.Field.JsonSchemaPrimitive;
-import java.io.IOException;
-import java.sql.Connection;
-import java.time.Instant;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
-import org.apache.commons.dbcp2.BasicDataSource;
-import org.apache.commons.lang3.tuple.ImmutablePair;
-import org.jooq.DSLContext;
-import org.jooq.DataType;
-import org.jooq.JSONFormat;
-import org.jooq.JSONFormat.RecordFormat;
-import org.jooq.Named;
 import org.jooq.SQLDialect;
-import org.jooq.Table;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class JdbcSource implements Source {
+public class JdbcSource extends AbstractJdbcSource implements Source {
```

Comment:
i like being able to run this source, but admittedly in its current implementation the concrete JdbcSource class is a little silly.